samples/ingestion/ingestion-client/Setup/guide.md

```diff
@@ -74,6 +74,11 @@ In real time mode, audio files are downloaded and streamed from the Azure Functi
 * **Only .wav PCM is supported.**
 * **You will need to use the [Real Time ARM Template](ArmTemplateRealtime.json) from the repository for this operating mode.**
 
+#### Scale up
+Batch mode will process transcription requests following best effort policies via which compute is requested as soon as the transcription is scheduled. Available compute is directly allocated. 
+
+In Real time mode, each Azure Speech resource is allocated by default 50 concurrent connections, which indicates the maximum number of parallel audio transcription streams. Customers do have a choice to raise a request to have that number increased through Azure Speech customer channels. To avoid throttling however the rhythm/tempo of uploading audio files to Azure storage (therefore triggering real time transcriptions) should be controlled. If it is not controlled it will result in throttling.
```

Comment:
```suggestion
In Real time mode, each Azure Speech resource is allocated with a default of 50 concurrent connections, which indicates the maximum number of parallel audio transcription streams. Customers can request a higher limit. To avoid throttling, the cadence of new audio file uploads to Azure storage should be controlled, because each upload triggers  a real-time transcription. Throttling occurs when the concurrency limit is reached.
```